packages/smart/contracts/turbo/AbstractMarketFactoryV3.sol

```diff
@@ -6,8 +6,9 @@ import "../libraries/IERC20Full.sol";
 import "../balancer/BPool.sol";
 import "./TurboShareTokenFactory.sol";
 import "./FeePot.sol";
+import "../libraries/Rewardable.sol";
 
-abstract contract AbstractMarketFactoryV3 is TurboShareTokenFactory, Ownable {
+abstract contract AbstractMarketFactoryV3 is TurboShareTokenFactory, Ownable, Rewardable {
```

Comment:
I wouldn't put this here. Instead, make each MarketFactory a Rewardable, if it can support rewards.